R/gdalvector.R

```diff
@@ -168,11 +168,17 @@
 #'
 #' \code{$returnGeomAs}\cr
 #' Character string specifying the return format of feature geometries.
-#' Must be one of `WKB` (the default), `WKB_ISO`, `WKT`, `WKT_ISO` or `NONE`.
+#' Must be one of `WKB` (the default), `WKB_ISO`, `WKT`, `WKT_ISO`, `BBOX`, or
+#' `NONE`.
 #' Using `WKB`/`WKT` exports as 99-402 extended dimension (Z) types for Point,
 #' LineString, Polygon, MultiPoint, MultiLineString, MultiPolygon and
 #' GeometryCollection. For other geometry types, it is equivalent to using
 #' `WKB_ISO`/`WKT_ISO` (see \url{https://libgeos.org/specifications/wkb/}).
+#' Using `BBOX` exports as a list of numeric vectors, each of length 4 with values
+#' `xmin, ymin, xmax, ymax`. If an empty geometry is encountered these values will
+#' `NA_real_` in the corresponding location.
```

Comment:
done

Reply:
eek, I didn't realize I had edited *your* comment above, here's what I'd said

> Oh I did not know you could construct rct that way, that's excellent, usually I pass the four columns in individually to `rct()`.  wk is always column-based afaik so that makes sense. (Oh I see, you can `as_xy()` a matrix  in the same way, otherwise `xy()` is separate columns  ...). rgl has a bunch of "reasonable assumptions" that work like this too and I really like that behaviour.